client/src/app/components/questionnaire-summary/questionnaire-summary.tsx

```diff
@@ -36,18 +36,16 @@ interface QuestionnaireSummaryProps {
   fetchError?: AxiosError | null;
   summaryData: Assessment | Questionnaire | undefined;
   summaryType: SummaryType;
-  isArchetype?: boolean;
 }
 
 const QuestionnaireSummary: React.FC<QuestionnaireSummaryProps> = ({
   summaryData,
   summaryType,
   isFetching = false,
   fetchError = null,
-  isArchetype,
 }) => {
   const { t } = useTranslation();
-
+  const isArchetype = location.pathname.includes("/archetypes/");
```

Comment:
Hm, what's the reason for changing this from a prop? If we ever needed to show the questionnaire content for an archetype outside the archetypes page (as a modal or something) this would be an issue. Not that we ever need to be able to do that necessarily, it's probably fine, just curious

Reply:
I changed everything to be driven off of the location path because I was seeing some undefined props on hard navigation to some of the routes. Maybe that would be better handled by redirects. Some more testing is probably warranted here.

Reply:
If you made the `isArchetype` prop required and provided a default value like `isArchetype = false`, then you would not see an undefined value.

Alternatively, you could use the `useLocation` or `useRouteMatch` hook from react-router to be a bit more robust in the `isArchetype` check.